src/flagpole/__init__.py

```diff
@@ -96,37 +97,60 @@ class Feature(BaseModel):
     name: constr(min_length=1, to_lower=True)  # type:ignore[valid-type]
     owner: constr(min_length=1)  # type:ignore[valid-type]
     segments: list[Segment]
+    """A list of segments to evaluate against the provided data"""
     enabled: bool = True
+    """Defines whether or not the feature is enabled."""
     created_at: datetime = Field(default_factory=datetime.now)
     """This datetime is when this instance was created. It can be used to decide when to re-read configuration data"""
+    context_builder: ContextBuilder | None = None
+    """Optional builder used to transform passed in data to an EvaluationContext whenever this feature is evaluated."""
```

Comment:
I'm not sure we want to have the context builder on `Feature`. Ideally we can share context generation between flagpole and flagr backends when we start integrating.

Reply:
Good point, I reverted this method back to taking an evaluation context instead.